app/experimenter/experiments/migrations/0194_auto_20211102_1749.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.5 on 2021-11-02 17:49
+
+from django.db import migrations
+
+
+def set_feature_value(apps, schema_editor):
+    NimbusBranch = apps.get_model("experiments", "NimbusBranch")
+    NimbusBranch.objects.filter(feature_value="").update(feature_value="{}")
```

Comment:
I'd guess this might fail JSON schema validation if we have any feature schemas with required keys? But should at least no longer cause the error to be thrown

Reply:
None of the schemas require keys, they're set that way on purpose